stories/Autocomplete.stories.js

```diff
@@ -77,7 +63,9 @@ storiesOf('Autocomplete', module)
     />
   ))
 
-  .add('Enter mnemonics (9-word mnemonic) - not sorted', withState({ selectedOpts: [] }, store => (
+  .add(
+    "(9-word mnemonic) - not sorted",
```

Comment:
Just a minor detail … but im not super happy with pushing the story description to the next line … looks weird to my eyes 😜